tasks/kmt.py

```diff
@@ -681,6 +684,15 @@ def prepare(
     for d in domains:
         d.copy(ctx, paths.dependencies, "/opt/", verbose=verbose)
         d.copy(ctx, f"{paths.arch_dir}/opt/*", "/opt/", exclude="*.ninja", verbose=verbose)
+        d.run_cmd(
+            ctx,
+            f"! [ -f {btf_dir}/minimized-btfs.tar.xz \
+                && [ -d /opt/btf ] \
+                && cd /opt/btf/ \
+                && tar cJf minimized-btfs.tar.xz * \
+                && mkdir -p {btf_dir} \
+                && mv /opt/btf/minimized-btfs.tar.xz {btf_dir}/",
```

Comment:
You don't actually need to create the `minimized-btfs.tar.xz` file. We can use the `.tar.xz` for the specific kernel. Let me get the path it needs to be at.

Reply:
Actually, looks like I'm wrong. We rely upon `minimized-btfs.tar.xz` to find the path, and then will check to see if we've already extracted it.